doc/tutorial/examples.txt

```diff
@@ -500,8 +500,8 @@ It will be used repeatedly.
   training_steps = 10000
   
   # Declare Theano symbolic variables
-  x = T.matrix("x")
-  y = T.vector("y")
+  x = T.dmatrix("x")
+  y = T.dvector("y")
```

Comment:
I fixed the other comments.

On Sat, Aug 8, 2015 at 6:51 PM, Frédéric Bastien <frederic.bastien@gmail.com
> wrote:

> This is before we talk about floatX, so I taught that forcing float64
> would make it easier to understand that example.
>
> On Sat, Aug 8, 2015 at 6:23 PM, Pascal Lamblin <notifications@github.com>
> wrote:
>
>> In doc/tutorial/examples.txt
>> <https://github.com/Theano/Theano/pull/3263#discussion_r36582470>:
>>
>> > @@ -500,8 +500,8 @@ It will be used repeatedly.
>> >    training_steps = 10000
>> >
>> >    # Declare Theano symbolic variables
>> > -  x = T.matrix("x")
>> > -  y = T.vector("y")
>> > +  x = T.dmatrix("x")
>> > +  y = T.dvector("y")
>>
>> If we want to make that example work with float32, it would be better to
>> get the code from the test instead:
>>
>> https://github.com/nouiz/Theano/blob/mixed/theano/tests/test_tutorial.py#L727-L776
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Theano/Theano/pull/3263/files#r36582470>.
>>
>
>